SPECS-EXTENDED/bsh/bsh.spec

```diff
@@ -60,7 +60,7 @@ BuildRequires:  ImageMagick
 BuildRequires:  desktop-file-utils
 %endif
 
-Requires:       java-headless
```

Comment:
I think I recall this issue being mentioned. Do you know why the team providing the Java packages cannot add the `java-headless` subpackage? Do we just want to avoid the delay with such request?

Reply:
An idea for an alternative solution: create our own dummy `java-headless` package, which `Requires: java`.

Reply:
Our Java folks have already approved a request for the appropriate "Provides" to be added to the `msopenjdk-11` RPM. We just want to avoid the delay, since the change will come as part of their update cycle (ETA later this month). As for converting the manual `java-headless` dependencies, I think it's fine in this case. Any spec manually requiring `java` or `java-headless` should really be converted to use the `javapackages-tools` package to generate that requirement.